v2/test/single_operator_multitenant_test.go

```diff
@@ -127,6 +127,7 @@ func Test_Multitenant_SingleOperator_PerResourceCredential(t *testing.T) {
 
 	objKey := client.ObjectKeyFromObject(acct)
 	tc.GetResource(objKey, acct)
+	tc.Expect(acct.Annotations).ToNot(HaveKey(reconcilers.PerResourceSecretAnnotation))
```

Comment:
oh wait! on line 116 above, the annotation has `nsName.String()` as a secret name which should get rejected as `.String()` method would add a separator `/`?

Reply:
Ahh good catch, will fix.